MailCore/Cache/TokenStore.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import InfomaniakCore
+import InfomaniakDI
+
+public class TokenStore {
+    public enum TokenStoreFetchLocation {
+        case cache
```

Comment:
Do we prefer 
```swift
case cache, keychain
```
or
```swift
case cache
case keychain
```
I noticed that I often write only one `case` but have no preference

Reply:
I do not have a preference either but I would tend to say that multiple line cases improve readability and are easier to refactor if we add more cases in the future.